src/core/time.d

```diff
@@ -355,9 +364,11 @@ version(Posix)
             case coarse: return CLOCK_MONOTONIC_COARSE;
             case normal: return CLOCK_MONOTONIC;
             case precise: return CLOCK_MONOTONIC;
-            case processCPUTime: return CLOCK_PROCESS_CPUTIME_ID;
             case raw: return CLOCK_MONOTONIC_RAW;
-            case threadCPUTime: return CLOCK_THREAD_CPUTIME_ID;
+            version(CRuntime_Glibc) {
+                case processCPUTime: return CLOCK_PROCESS_CPUTIME_ID;
+                case threadCPUTime: return CLOCK_THREAD_CPUTIME_ID;
+            }
```

Comment:
These won't be defined for Bionic anymore, and [they appear to be defined for Musl](http://git.musl-libc.org/cgit/musl/tree/include/time.h?h=v1.1.18#n87)?